src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -2803,12 +2803,12 @@ internal void EnsureAnonymousTypeTemplates(CancellationToken cancellationToken)
                 }
                 catch (Cci.PeWritingException e)
                 {
-                    diagnostics.Add(MessageProvider.CreateDiagnostic(MessageProvider.ERR_PeWritingFailure, Location.None, e.InnerException.ToString()));
+                    diagnostics.Add(MessageProvider.CreateDiagnostic(MessageProvider.ERR_PeWritingFailure, Location.None, e.InnerException?.ToString() ?? ""));
```

Comment:
nit: just curious, is there a difference between using `""` and `string.Empty`? The PR uses both in different places.

Reply:
I originally used `""` here but @cston pointed out this file prefers `string.Empty` hence I changed to be consistent within this file. So basically trading PR consistency for file consistency 😄